airquality/README.md

```diff
@@ -0,0 +1,65 @@
+Provided by: Kristina Robinson
+
+Air Quality (Ozone and Small Particulates per Million) by zipcode from the U.S. EPA AirNow.gov API.
+
+Quick Install
+
+InfluxDB UI
+
+In the InfluxDB UI, go to Settings-> Templates and enter this URL: http://raw.githubusercontent.com/influxdata.community-templates.master/airquality/airquality.yaml
```

Comment:
This URL doesn't look right, it should be the same as the one in the CLI instructions below